src/main/java/org/sonarlint/intellij/ui/SonarLintRulePanel.java

```diff
@@ -226,6 +263,53 @@ private static String fixPreformatedText(String htmlBody) {
     return builder.toString();
   }
 
+  private String renderRuleParams(StandaloneRuleDetails ruleDetails) {
+    if (!ruleDetails.paramDetails().isEmpty()) {
+      StyleSheet styleSheet = kit.getStyleSheet();
+
+      styleSheet.addRule(".rule-params { border: none; border-collapse: collapse; padding: 1em; }");
+      styleSheet.addRule(".rule-params caption { font-size: 16px; font-weight: 400; text-align: left; margin-bottom: 16px}");
+      styleSheet.addRule(".rule-params thead td { vertical-align: top; padding-left: 0; padding-bottom: 1em; font-style: italic }");
```

Comment:
It looks like some of these styles are not applied, maybe we can do something more visually appealing within the constraints of the Swing HTML renderer.

![image](https://user-images.githubusercontent.com/1734254/89168600-6035fa80-d57d-11ea-8c1b-cb78faaaef26.png)